guides/troubleshooting/inotify-watch-limits.md

```diff
@@ -262,7 +262,7 @@ program sets the `fs.inotify.max_user_watches` setting.
   protocol for programming language support, and the same constraints and
   limitations apply to those tools)
 - Resources for Visual Studio Code and code-server:
-  - [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings),
+  - [User and workspace Settings](https://code.visualstudio.com/docs/getstarted/settings),
```

Comment:
Since this links out to their page not sure if the w should be lowercased. If we do want to lowercase it, then the s in settings should also be lowercased.